packages/backend/src/webhook/service.ts

```diff
@@ -110,19 +110,23 @@ async function sendWebhookEvent(
       processAt: null
     })
   } catch (err) {
+    const error = err as AxiosError
```

Comment:
I think we can use `isAxiosError` typeguard from `axios` to make this a bit safer